tests/fixtures/decam.py

```diff
@@ -149,8 +158,8 @@ def decam_raw_origin_exposures():
 
 
 @pytest.fixture(scope="session")
-def decam_filename(data_dir, cache_dir):
-    """Pull a DECam exposure down from the NOIRLab archives.
+def decam_filename(download_url, data_dir, decam_cache_dir):
+    """Pull a DECam exposure down from the NOIRLab archives.  TODO: should we download from NERSC or from NOIRLab?
```

Comment:
Re: the TODO, that depends what we want to test.  If we want to test that we know how to pull from the NOIRLab archive, leave it that way.  If we don't care, and just want to get the exposure, it will almost certainly perform better where we will be running most of the time if we pull it from NERSC.

Reply:
I will make the fixture pull from our URL and add a "slow" test for downloading from NOIRLab.